ercc/attestation/provider.go.template

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
```

Comment:
I guess you didn't use `git mv` when moving this file? (Luckily you seem to have done for the other plugin files you moved where retaining the history is a bit more crucial ...)

Reply:
uuhh ... nice catch! I can fix that once I squashed the fixups.

Reply:
if it is not much effort, then this would be nice. otherwise probably not worth. This might anyway go away sooner or later? (e.g., iff we have to resort to externalbuilder for ercc because we have c-code in it, then we might as well also use build-tags?)

Reply:
ok - let's keep it.